.circleci/config.yml

```diff
@@ -13,7 +13,11 @@ workflows:
           name: node-latest
           node_version: latest
       - release-management/test-package:
-          name: node-12
+          name: node-lts-active
+          node_version: '14'
+      - release-management/test-package:
+          name: node-lts-maintenance
+          node_version: '12'
```

Comment:
What do you think of adopting something [like this](https://github.com/salesforcecli/plugin-auth/pull/80/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R12-R20) for unit testing going forward?

Reply:
This requires a change each time new versions of node are created. Is there a simple way to supply symbolic names that are already well known or can be set from a single location.

Reply:
You would have to have another condition "if version == 'lts'" then add `--lts` to the nvm install command. See nvm's readme.